### PR TITLE
added date GE rule

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: tests/data/mock_manifests/Invalid_Test_Manifest.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Component,Check List,Check Regex List,Check Regex Format,Check Regex Single,Check Num,Check Float,Check Int,Check String,Check URL,Check Match at Least,Check Match at Least values,Check Match Exactly,Check Match Exactly values,Check Recommended,Check Ages,Check Unique,Check Range,Check Date
-MockComponent,"ab,cd","ab,cd,ef",a,a,6,99.65,7,valid,https://www.google.com/,1738,1738,8085,98085,,6549,str1,70,32984
+MockComponent,"ab,cd","ab,cd,ef",a,a,6,99.65,7,valid,https://www.google.com/,1738,1738,8085,98085,,6549,str1,70,32-984
```
Comment (GiaJordan) [inline tests/data/mock_manifests/Invalid_Test_Manifest.csv:2]: Can you add assertions in `test_validation.TestManifestValidation.test_invalid_manifest` that confirm the appropriate error messages are generated?

File: schematic/models/GE_Helpers.py [hunk 2]
```diff
@@ -402,6 +418,7 @@ def generate_errors(
                 # indices and values cannot be returned
                 else:
                     for i, item in enumerate(self.manifest[errColumn]):
+                        logging.warning(result_dict)
```
Comment (GiaJordan) [inline schematic/models/GE_Helpers.py:421]: Can this be removed?

File: pyproject.toml [hunk 3]
```diff
@@ -69,6 +69,7 @@ Flask-Cors = "^3.0.10"
 pdoc = "^12.2.0"
 dateparser = "^1.1.4"
 schematic-db = {version = "^0.0.6", extras = ["synapse"]}
+pyopenssl = "^23.0.0"
```
Comment (GiaJordan) [inline pyproject.toml:72]: This will be handled by the other PR right?

**Review by GiaJordan — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (andrewelamb) [reply pyproject.toml:72]: Yes

File: tests/test_validation.py [hunk 4]
```diff
@@ -65,6 +65,8 @@ def test_invalid_manifest(self,helpers,sg,metadataModel):
             rootNode=rootNode
             )
 
+        logging.warning(errors)
```
Comment (GiaJordan) [inline tests/test_validation.py:68]: Can you remove this `logging` as well, and add a description to the PR?

**Review by GiaJordan — APPROVED** [review]

**Pull request merged**